force-app/main/default/lwc/rd2StatusAutomationSettings/rd2StatusAutomationSettings.js

```diff
@@ -188,15 +185,23 @@ export default class Rd2StatusAutomationSettings extends LightningElement {
         this.adjustDaysForClosedErrorMessage();
     }
 
+    assignInputStatusWithApiValue(automationSettings) {
+        this.inputClosedStatus = automationSettings.closedStatusOption.filter(
```

Comment:
Pretty slick!

Reply:
I had to google how to use a for loop in Javascript and this popped out.